recipes/gcluster/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "Gcluster" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "0c893a9ddbbb91919be0eeae5212d1e695b811c049619ba56d805a6e52087bad" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+    url: https://github.com/Xiangyang1984/Gcluster/archive/Gcluster-v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - mcl
+    - blast
+    - perl-gd
+    - perl-gd-svg
+    - perl-svg
+    - perl-bioperl-core
+    - git
```

Comment:
Does this really need `git` to run? It's fine if so, it's just an unusual requirement.

Reply:
@dpryan79 please remove git for me, thanks very much.